examples/sdk-examples/src/examples/export/UseDataExportExample.tsx

```diff
@@ -0,0 +1,66 @@
+// (C) 2007-2022 GoodData Corporation
+import React, { useState, useCallback, useEffect } from "react";
+import { useDataExport } from "@gooddata/sdk-ui";
+import { IExportConfig, IPreparedExecution } from "@gooddata/sdk-backend-spi";
+
+import { useBackend } from "../../context/auth";
+import { workspace } from "../../constants/fixtures";
+import * as Md from "../../md/full";
+
+import { Button, downloadFile } from "./ExampleWithExport";
```

Comment:
These imports should be inlined so you can see the code immediately in the example source code